server/services/reportService.ts

```diff
@@ -63,6 +73,20 @@ export default class ReportService {
     return report
   }
 
+  async getReportInProgress(bookingId: number): Promise<inProgressReport | null> {
+    const report = await this.incidentClient.getReportInProgress(bookingId, ReportStatus.IN_PROGRESS.value)
+    return report
+  }
+
+  async getReportsByDate(bookingId: number, incidentDate: string): Promise<offenderReports[] | []> {
```

Comment:
Have used a string parameter for incidentDate so that I can create a moment + 1 day.
The logic behind + 1 day, need to get all the reports for a specific incident date between 00:00 and 24:00.
This achieved by calling the DB  with date >= date 00h:00m and date < date + 1 00h:00m
Setting date and date +1 could have been done in the controller but then it would have tied the getReportsByDate too closely to the controller.

Reply:
it would better to pass through the date as a correctly typed `Moment` - should enforce type-safety at boundaries. 

Think you should be able to work out the range using: 
```
moment(incidentDate).startOf('day')
moment(incidentDate).endOf('day')
```

Reply:
done. incidentDate is now the Date returned by draftReportService.getCurrentDraft